mobile/src/main/java/org/openhab/habdroid/ui/CloudNotificationAdapter.java

```diff
@@ -154,7 +155,11 @@ public void bind(CloudNotification notification) {
                 Connection conn = ConnectionFactory.getConnection(Connection.TYPE_CLOUD);
                 String iconUrl = String.format(Locale.US, "images/%s.png",
                         Uri.encode(notification.icon()));
-                mIconView.setImageUrl(conn, iconUrl, 2000);
+                mIconView.setImageUrl(
+                        Util.convertDpToPixel(
+                                mIconView.getResources().getDimension(R.dimen.notificationlist_icon_size),
+                                mIconView.getContext()),
```

Comment:
`mIconView.setImageUrl(mIconView.getResources.getDimensionPixelSize(R.dimen.notificationlist_icon_size), conn, iconUrl, 2000);`

(`getDimensionPixelSize` returns, unless I'm utterly mistaken, pixel values)

Reply:
I didn't know about this function. I'll change it and remove both convert functions in Util.java.